packages/generator-particle-patternlab/src/generators/app/index.ts

```diff
@@ -0,0 +1,49 @@
+import Generator from 'yeoman-generator'
+import fs from 'fs'
+import _ from 'lodash'
```

Comment:
Let's use the lodash/merge npm package. Check the particle-base generator for that dependency.

Reply:
https://www.npmjs.com/package/lodash.merge